crates/uv/src/commands/tool/install.rs

```diff
@@ -373,5 +374,51 @@ pub(crate) async fn install(
     );
     installed_tools.add_tool_receipt(&from.name, tool)?;
 
+    // If the executable directory isn't on the user's PATH, warn.
+    if !std::env::var_os("X")
+        .as_ref()
+        .iter()
+        .flat_map(std::env::split_paths)
+        .any(|path| same_file::is_same_file(&executable_directory, path).unwrap_or(false))
+    {
+        let export = match Shell::from_env() {
+            None => None,
+            Some(Shell::Nushell) => None,
```

Comment:
Couldn't find a short invocation for this, it looks like you have to modify a file and source it.